pkg/git/cloner.go

```diff
@@ -29,22 +30,47 @@ import (
 	"gopkg.in/src-d/go-git.v4/plumbing/transport/http"
 )
 
-//ReposPerProject returns all the repositories contained within a github project
+//ReposPerProject returns all the repositories contained withina  bitbucket or github project
```

Comment:
We should probably remove the BB stuff and just keep it specific to github? THoughts?

Reply:
I don't think there's any harm to having it here - people do use bitbucket so they'd be able to leverage earlybird if they do. It also stands as an example that we can support multiple SCM systems so if anyone wanted to extend functionality for gitlab for example they could add that here. What do you think?

Reply:
yah then should be ok... also a type needs fixing withina should be -> `within a`